doc/man7/EVP_KDF-ARGON2.pod

```diff
@@ -0,0 +1,169 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF-ARGON2 - The Argon2 EVP KDF implementation
+
+=head1 DESCRIPTION
+
+Support for computing the B<argon2> password-based KDF through the B<EVP_KDF>
+API.
+
+The EVP_KDF-ARGON2 algorithm implements the Argon2 password-based key
+derivation function, as described in IETF RFC 9106.  It is memory-hard in
+the sense that it deliberately requires a significant amount of RAM for efficient
+computation. The intention of this is to render brute forcing of passwords on
+systems that lack large amounts of main memory (such as GPUs or ASICs)
+computationally infeasible.
```

Comment:
I am missing at least short mention of the ARGON2I/D/ID variants and at least some minimal description of the differences.

Reply:
Added:
```
Argon2d (Argon2i) uses data-dependent (data-independent) memory access and primary seek to address trade-off (side-channel) attacks.

Argon2id is a hybrid construction which, in the first two slices of the first pass, generates reference addresses data-independently as in Argon2i, whereas in in later slices and next passess it generates them data-dependently as in Argon2d.

Sbox-hardened version Argon2ds is not supported.

For more information, please refer to RFC 9106.
```

Does that work?

Reply:
LGTM